Regression/WarpX-tests.ini

```diff
@@ -163,6 +163,25 @@ aux1File = Tools/PostProcessing/read_raw_data.py
 analysisRoutine = Examples/Modules/boosted_diags/analysis_3Dbacktransformed_diag.py
 tolerance = 1.e-14
 
+[Reformatted_BTD_ReducedSliceDiag]
```

Comment:
If we turn this test off, maybe we should create an issue to keep track of the fact that we need to further split up the tests into more CI runners.